src/modules/ROOT/pages/changelog/202209/06.json

```diff
@@ -0,0 +1,206 @@
+  },
+
+  "changed": [
+    {
+      "what": [
+        "workflow iterators to treat limit 0 parameter as no limit set."
+      ],
+      "why": [
+        "to make workflow iterators consistent with other searching/filtering interactions."
+      ],
+      "how": [
+        "by changing the limit in the code."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/fc259ca62"
+      ]
+    },
+    {
+      "what": [
+        "error message when trying to delete a sensitivity level that's in use to be more informative on what is wrong."
+      ],
+      "why": [
+        "to make error messages more consistent and user friendly."
+      ],
+      "how": [
+        "by adding additional validation before the sensitivity level is deleted."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/56e55439d"
+      ]
+    },
+    {
+      "what": [
+        "the closing of the import modal."
+      ],
+      "why": [
+        "to give the user a better UX."
+      ],
+      "how": [
+        "by adding an id to the import modal through which the modal is closed after a successful import."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/735/commits/9931190db6eb928b8e9ebe9819d9c22e58d42f7c"
+      ]
+    },
+    {
+      "what": [
+        "the location of the attachment modal."     
+      ],
+      "why": [
+        "to enable previewing of attachments in private pages."
+      ],
+      "how": [
+        "by moving the attachment modal from Public/Pages/View to Namespace/View."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/ea1c1fa7889b3efb83e94127fcaad4ac44e56a2b" 
+      ]
+    },
+    {
+      "what": [
+        "the requirement to provide a handle when creating a role is now removed and the handle is now optional."
+      ],
+      "why": [
+        "because providing a handle shouldn't be a necessity.",
+        "The resource can be created without it."
+      ],
+      "how": [
+        "by removing the requirement for the handle to be provided."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/fd55cce9b" 
+      ]
+    }
+  ],
+
+  "fixed": [
+    {
+      "what": [
+        "invalid record revision history for multi value fields."        
+      ],
+      "how": [
+        "The first value appeared duplicated, the last value was omitted, and when a value was removed the value was no longer visible in the old version."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/b203224df"
+      ]
+    },
+    {
+      "what": [
+        "uninformative internal error error message on the user profile change password interface."        
+      ],
+      "how": [
+        "by properly handling and serving the error message."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/82c70b366"
```

Comment:
I'd consider replacing this more technical explanation with something easier to understand or remove it altogether (or incorporate into what).